Assignment/solution_student/css/style.css

```diff
@@ -6,18 +6,175 @@ Developer: @poopsplat in Chicago
 */
 
 /******************************************
+/* SETUP
+/*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+/******************************************
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
-  font-size: 16px;
-  background: white;
+  	color: #121212;
+  	font-size: 16px;
+  	font-weight: 300;
+  	margin-top: 0px;
+}
+
+h1{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 24px;
+	font-weight: 800;
+	text-align: left;
+	color: #FFF;
+	padding: 25px 10px 10px 10%;
+}
+
+h2{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 24px;
+	font-weight: 500;
+	color: #FFF;
+}
+
+h3{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 20px;
+	font-weight: 500;
+	color: #033048;
+}
+
+h4{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 24px;
+	font-weight: 800;
+	text-align: center;
+	color: #f9e42e;
+}
+
+h5{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 18px;
+	font-weight: 500;
+	text-align: center;
+	color: #fff;
+}
+
+a{
+	font-family: 'Open Sans', sans-serif;
+	font-size: 14px;
+	font-weight: 800;
+	border-radius: 5px; 
 }
 
+
 /******************************************
 /* LAYOUT
 /*******************************************/
+.container{
+	padding: 0px;
+	width: 80%;
+	margin: auto;
+	margin-top: 0px;
+
+}
+
+.header{
+ 	text-align: center;
+ 	background-image: url(../images/header_bg.jpg);
+ 	background-size: 100%;
+ 	background-repeat: no-repeat;
+ 	padding: 0px;
+ 	margin: 0px;
+ 	height: 450px;
+}
 
+.main{
+	text-align: 0px;
+	background-color: #f0efef;
+	margin: 0px;
+	padding: 30px 40% 40px 60px;
```

Comment:
Another route you could have gone instead of the `40%` padding, is to wrap the contents of the `.main` div with a container div, and then give that  `max-width` and `margin: 0 auto;` to keep it centered:
```
<div class="main".
  <div class="main-container">
    <!-- the contents -->
  </div>
</div>
```
...and then in your CSS:
```
.main-container {
  max-width: 1200px; /* or whatever works best */
  margin: 0 auto; /* keep it centered */
}
```
That way it doesn't get too squished at smaller-sized screens (thinking ahead to our responsive lessons), and you can maintain a consistent padding on all sides. Just a suggestion though!